CODEOWNERS

```diff
@@ -1,41 +1,158 @@
+packages/app-bridge/src/react/useTemplateAssets.ts @Frontify/guidelines-navigation-themes
+packages/app-bridge/src/react/useTemplateChooser.spec.tsx @Frontify/guidelines-one
+packages/app-bridge/src/react/useTemplateChooser.ts @Frontify/guidelines-one
+packages/app-bridge/src/react/useThemeAssets.spec.ts @Frontify/guidelines-navigation-themes
+packages/app-bridge/src/react/useThemeAssets.ts @Frontify/guidelines-navigation-themes
+packages/app-bridge/src/react/useThemeSettings.spec.ts @Frontify/guidelines-navigation-themes
+packages/app-bridge/src/react/useThemeSettings.ts @Frontify/guidelines-navigation-themes
+packages/app-bridge/src/react/useUncategorizedDocumentPages.spec.ts @Frontify/guidelines-one @Frontify/guidelines-navigation-themes
+packages/app-bridge/src/react/useUncategorizedDocumentPages.ts @Frontify/guidelines-one @Frontify/guidelines-navigation-themes
+packages/app-bridge/src/react/useUngroupedDocuments.spec.ts @Frontify/guidelines-one @Frontify/guidelines-navigation-themes
+packages/app-bridge/src/react/useUngroupedDocuments.ts @Frontify/guidelines-one @Frontify/guidelines-navigation-themes
+
+### App Bridge registries api
+packages/app-bridge/src/registries/api/CreateAsset.ts @Frontify/integration
+packages/app-bridge/src/registries/api/CreateAsset.spec.ts @Frontify/integration
+packages/app-bridge/src/registries/api/GetAssetResourceInformation.ts @Frontify/integration
+packages/app-bridge/src/registries/api/GetAssetResourceInformation.spec.ts @Frontify/integration
+packages/app-bridge/src/registries/api/GetAssetBulkDownloadToken.ts @Frontify/guidelines-one
+packages/app-bridge/src/registries/api/GetCurrentUser.ts @Frontify/guidelines-one @Frontify/integration @Frontify/guidelines-navigation-themes
+packages/app-bridge/src/registries/api/GetCurrentUser.spec.ts @Frontify/guidelines-one @Frontify/integration @Frontify/guidelines-navigation-themes
+
+### App Bridge registries commands
+packages/app-bridge/src/registries/commands/AssetChooser.ts @Frontify/guidelines-one @Frontify/guidelines-navigation-themes
+packages/app-bridge/src/registries/commands/AssetViewer.ts @Frontify/guidelines-one
+packages/app-bridge/src/registries/commands/Connection.ts @Frontify/integration
+packages/app-bridge/src/registries/commands/DownloadAsset.ts @Frontify/guidelines-one
+packages/app-bridge/src/registries/commands/NavigationManager.ts @Frontify/guidelines-navigation-themes
+packages/app-bridge/src/registries/commands/NewPublication.ts @Frontify/guidelines-one
+packages/app-bridge/src/registries/commands/TemplateChooser.ts @Frontify/guidelines-one
+
+### App Bridge registries events
+packages/app-bridge/src/registries/events/ @Frontify/guidelines-one @Frontify/guidelines-navigation-themes
+
+### App Bridge registries verbs
+packages/app-bridge/src/registries/verbs/ @Frontify/guidelines-one @Frontify/guidelines-navigation-themes @Frontify/integration
+
+## App Bridge repositories
+packages/app-bridge/src/repositories/ @Frontify/guidelines-one
```

Comment:
@ragi96 Should all the repositories be owned just by guidelines 1? Or is this just for now and we can split it further when the time is ready?

Reply:
I think in the long run this `repositories` should be moved out of the app-bridge so we can control it within the web-app. Atm there is just one Repo there and its only used by G1, so if something gets added (i dont think so), it will be splited further

Reply:
Ah perfect, thanks for the explanation!